sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -132,6 +132,12 @@ Licensed under the MIT License.
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-serializer-json-jackson</artifactId>
+      <version>1.0.0-beta.1</version>
```

Comment:
Missing version tag

Reply:
Will add version tags when the PR is closer to being ready for review.